samples/net/cloud/aws_mqtt/src/certificates.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * openssl x509 -in <filename>.pem -inform pem -noout -C
+ */
+
+#error "Fill this before generating the build"
+
+unsigned char amazon_certificate[] = {
+};
+
+
+unsigned char private_key[] = {
```

Comment:
Is this key expected to be filled in by someone? Please look at the `samples/net/cloud/google_iot_mqtt` to see how the keys are handled there. Right now, it isn't obvious how this would be loaded (or that it needs to be).